src/gitops/components/application/ApplicationSetDetailsPage.tsx

```diff
@@ -0,0 +1,636 @@
+                                    </div>
+                                    <div style={{ flex: 2, textAlign: 'left' }}>{condition.reason || ''}</div>
+                                    <div style={{ flex: 4, textAlign: 'left' }}>{condition.message || ''}</div>
+                                  </div>
+                                  {index !== status.conditions.length - 1 && (
+                                    <div style={{ borderTop: '1px solid #393F44' }} />
+                                  )}
+                                </React.Fragment>
+                              ))}
+                            </div>
+                          </div>
+                        )}
+                      </CardBody>
+                    </Card>
+                  </div>
+                </div>
+              </div>
+            </Tab>
+
+            <Tab eventKey={1} title={<TabTitleText>YAML</TabTitleText>} className="pf-v6-c-tab-content">
+              <div className="co-m-pane__body">
+                <div className="co-yaml-editor">
+                  <div className="co-yaml-editor__header">
+                    <div className="co-yaml-editor__header-buttons">
+                      <button className="btn" aria-label="File">
+                        <i className="fa fa-file"></i>
+                      </button>
+                      <button className="btn" aria-label="Settings">
+                        <i className="fa fa-cog"></i>
+                      </button>
+                      <button className="btn" aria-label="Full screen">
+                        <i className="fa fa-expand"></i>
+                      </button>
+                    </div>
+                    <div className="co-yaml-editor__header-shortcuts">
+                      <a href="#" className="co-yaml-editor__header-shortcuts-link">Shortcuts</a>
+                    </div>
+                  </div>
+                  <div className="co-yaml-editor__content" style={{ background: '#1e1e1e', color: '#d4d4d4', fontFamily: 'monospace', fontSize: 14, borderRadius: 4, padding: 0 }}>
+                    <pre style={{ margin: 0, padding: 16, overflow: 'auto' }}>{JSON.stringify(appSet, null, 2)}</pre>
+                  </div>
+                </div>
+              </div>
+            </Tab>
+
+            <Tab eventKey={2} title={<TabTitleText>Generators</TabTitleText>} className="pf-v6-c-tab-content">
+              <div className="co-m-pane__body">
+                <div className="pf-v6-l-grid pf-m-gutter">
+                  <div className="pf-v6-l-grid__item pf-m-12-col-on-md">
+                    <Card>
+                      <CardHeader>
+                        <CardTitle>Generators</CardTitle>
+                      </CardHeader>
+                      <CardBody>
+                        <div style={{ display: 'flex', flexDirection: 'column', gap: '16px' }}>
+                          {appSet.spec?.generators?.map((generator: any, index: number) => {
```

Comment:
For an appset with a matrix generator, I don't see the individual generators listed under this tab.